app/views/propertypensionsinvestments/UkInsuranceGainsAgentView.scala.html

```diff
@@ -34,14 +55,22 @@
             @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "UkInsuranceGains")))
         }
 
+        @heading(messages("ukInsuranceGains.title"))
```

Comment:
Maybe we should use the agent key for the agent view even though they are the same value.

Reply:
Agree